### PR TITLE
Py enhancements

### PR DESCRIPTION
**Pull request by kmvanbrunt**
Various minor improvements, primarily with help messages and docstrings.

### CODE REVIEW

File: tests/test_cmd2.py [hunk 1]
```diff
@@ -1860,22 +1860,6 @@ def test_alias_create(base_app, capsys):
     out = run_cmd(base_app, 'alias list fake')
     assert out == normalize('alias create fake pyscript')
 
-def test_alias_quoted_name(base_app, capsys):
```
Comment (tleonhardt) [inline tests/test_cmd2.py]: Somehow removing these unit tests and/or changing code elsewhere in this PR has resulted in an uncovered line in **parsing.py** when ``_is_valid_command()`` is called with an empty string.  Please add a simple unit test for this other case.  I'd like to keep the coverage of **parsing.py** at 100% given its critical underlying role.

File: cmd2/cmd2.py [hunk 2]
```diff
@@ -2060,15 +2060,14 @@ def default(self, statement: Statement) -> None:
...
         if self.default_to_shell:
-            result = os.system(arg)
+            result = os.system(statement.command_and_args)
             # If os.system() succeeded, then don't print warning about unknown command
             if not result:
                 return
 
         # Print out a message stating this is an unknown command
-        self.poutput('*** Unknown syntax: {}\n'.format(arg))
+        self.poutput('*** Unknown syntax: {}\n'.format(statement.command_and_args))
```
Comment (tleonhardt) [inline cmd2/cmd2.py]: Nice improvements throughout.

File: tests/test_cmd2.py (hunk 1 above)
Comment (kmvanbrunt) [reply tests/test_cmd2.py]: Done

File: tests/test_parsing.py [hunk 3]
```diff
@@ -756,6 +756,38 @@ def test_statement_is_immutable():
         statement.raw = 'baz'
 
 
+def test_is_valid_command_invalid(parser):
```
Comment (tleonhardt) [inline tests/test_parsing.py]: Thanks for adding the unit test

**Pull request merged**